docs/support.rst

```diff
@@ -1,22 +1,44 @@
 Support
 =======
 
-Getting Help
-------------
+Usage Questions
+----------------
 
-The easiest way to get help with the project is to join the ``#readthedocs``
-channel on Freenode_. We hang out there and you can get real-time help with
-your projects.  The other good way is to open an issue on GitHub_.
+If you have questions about how to use Read the Docs well,
+`Stack Overflow`_ is the best place to ask. Tag questions with ``read-the-docs`` so other folks can find them easily.
 
-.. _Freenode: irc://freenode.net/
-.. _GitHub: http://github.com/rtfd/readthedocs.org/issues
+Community Support
+-----------------
 
-Backwards Incompatible Changes
-------------------------------
+Read the Docs is a community supported site.
+Currently nobody is paid to handle readthedocs.org support.
+We are hoping to bring in enough money with our `Gold`_ program to change that,
+so please sign up if you are able.
 
-We are thinking about having some kind of Backwards
-Incompatible Changes policy, much like the 1.0 of a code base, once we define
-the redirects and interfaces that we wish to expose permanently.
+**All people answering your questions are doing it with their own time,
+so please be kind and provide as much infromation as possible.**
+
+Bugs & Support Issues
+~~~~~~~~~~~~~~~~~~~~~
+
+You can file a bug on our `GitHub Issue Tracker`_,
+and it will be taken care of as soon as possible.
+However,
+there is no guaranteed response time,
+because everyone doing support is a volunteer.
```

Comment:
Sounds a little funny, maybe: "Support is a volunteer effort however, and there is no guaranteed response time."?